sources/oracle/toddl.go

```diff
@@ -99,6 +99,10 @@ func toSpannerTypeInternal(conv *internal.Conv, id string, mods []int64) (ddl.Ty
 		return ddl.Type{Name: ddl.String, Len: ddl.MaxLength}, nil
 	case "XMLTYPE":
 		return ddl.Type{Name: ddl.String, Len: ddl.MaxLength}, nil
+	case "JSON":
+		return ddl.Type{Name: ddl.JSON}, nil
+	case "ARRAY":
+		return ddl.Type{Name: ddl.String, Len: ddl.MaxLength, IsArray: true}, nil
```

Comment:
So oracle only support string arrays?

Reply:
Arrays in oracle are of custom types. And the underlying type is not restricted to string. It can support other types as well. To generalize it we mapped them to string array.